_docs/Qabel-Protocol-Box.md

```diff
@@ -281,6 +285,19 @@ url: STR // URL to the metadata file that contains information about the folder
 },
 ```
 
+ExternalFile:
+
+```
+{
+owner: STR, // public key of the owner of that File
+name: STR, // object name,
+size: LONG, // uncompressed file size
+mtime: LONG, // modification time as seconds since epoch
```

Comment:
The FM in the creators prefix is automatically updated during "Updating a file". Isn't it sufficient to only store owner, meta, block and maybe name and request current size and modification time from the FM?